Dockerfile

```diff
@@ -1,222 +1,8 @@
-# these need to be defined before any FROM - otherwise, the ARGs expand to empty strings.
-
-# pyspy & rbspy, using the same builder for both pyspy and rbspy since they share build dependencies - rust:1.59-alpine3.15
-ARG RUST_BUILDER_VERSION=@sha256:65b63b7d003f7a492cc8e550a4830aaa1f4155b74387549a82985c8efb3d0e88
-# pyperf - ubuntu 20.04
-ARG PYPERF_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
-# perf - ubuntu:16.04
-ARG PERF_BUILDER_UBUNTU=@sha256:d7bb0589725587f2f67d0340edb81fd1fcba6c5f38166639cf2a252c939aa30c
-# phpspy - ubuntu:20.04
-ARG PHPSPY_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
-# dotnet builder - mcr.microsoft.com/dotnet/sdk:6.0-focal
-ARG DOTNET_BUILDER=@sha256:749439ff7a431ab4bc38d43cea453dff9ae1ed89a707c318b5082f9b2b25fa22
-# async-profiler glibc build
-# requires CentOS 7 so the built DSO can be loaded into machines running with old glibc (tested up to centos:6),
-# we do make some modifications to the selected versioned symbols so that we don't use anything from >2.12 (what centos:6
```

Comment:
Please move this comment to its appropriate location in `build_x86_64_executable.sh`, I don't want this information forgotten.